retroshare-gui/src/gui/MainWindow.cpp

```diff
@@ -998,6 +998,13 @@ void SetForegroundWindowInternal(HWND hWnd)
 			_instance->ui->stackPages->setCurrentPage( _instance->postedDialog );
 			return true ;
 		 default:
+
+#ifdef RS_USE_WIRE
```

Comment:
normaly you add this before **default:**

![image](https://github.com/RetroShare/RetroShare/assets/9952056/00ce9d7e-13c0-4981-8e2d-4d39903c6e25)

Reply:
yea, i missed that, i will correct it.